extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -338,6 +339,17 @@ export class SubscriptionService {
     });
   }
 
+  /**
+   * Logs error and sends message to viewer
+   * @param {string} error
+   */
+  sendAuthTokenErrorToViewer_(error) {
+    this.viewer_.sendMessageAwaitResponse('auth-rejected', dict({
```

Comment:
Just `sendMessage`. No reason to wait for the response.

Reply:
Done